### DISCUSSION TITLE
Adding text to a video in lossless cut without using the subtitles

### DISCUSSION THREAD

**Discussion by kaizenakira** [mifi/lossless-cut#2170] [Support]
Is there a way to manually add text to a video in lossless cut without using the subtitles features?

Thank you for any help with this.

**Comment by mifi** [comment]
Not sure what you mean by text. But see faq about burn in subtitles

**Comment by kaizenakira** [comment]
Thank you so much for the info, I really appreciate it.

Michael Terao, M.D., M.Ed.
Cell: 301-661-0080
Personal Fax: 866-265-4314
